src/fldeff_teleport.c

```diff
@@ -0,0 +1,55 @@
+#include "global.h"
+#include "field_effect.h"
+#include "task.h"
+
+extern bool8 is_light_level_1_2_3_or_6(u8 mapType);
+extern void* hm_add_c3_launch_phase_2(void);
+extern void hm_teleport_run_dp02scr(void);
```

Comment:
Shouldn't be declared as extern as it's a forward declaration; also separate forward declarations from externs visually